src/gcp_scanner/crawl.py

```diff
@@ -495,8 +491,8 @@ def get_sql_instances(project_name: str,
     request = service.instances().list(project=project_name)
     while request is not None:
       response = request.execute()
-      for database_instance in response.get("items", []):
-        sql_instances_list.append(database_instance)
+      sql_instances_list = [database_instance 
+                            for database_instance in response.get("items", [])]
```

Comment:
This is redundant. Replace with `sql_instances_list = response.get("items", [])`

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.